### DISCUSSION TITLE
New rules for undead

### DISCUSSION THREAD

**Discussion by techred32** [commandgeek/geeksmp#75] [Ideas]
### What and How?

I think that there should be 2 new rules for undeads.
1. No spawn camping around alive players
2. No repetitively killing alive players

### Why?

I think we should do this so undeads like Bax and Daniel cannot repetitively kill alive players. When they were doing that the server wasn't fun and upsetting that everything was lost over and over. This should be able to prevent it.

### Are there alternatives?

There most likely are but I think this is the easiest and simplest for now.

### Checks

- [X] I have used the search at least once to check if my idea has already been suggested and perhaps already implemented.

**Comment by Lianecx** [comment]
GeekSMP is a pvp server: Player vs. undeads

**Reply by DWAA1660** [reply]
then allow us to bedtrap undeads since its pvp server
its really hard to fight back since we have rules and they dont have rules maybe if we have no rules against undeads it would be fair game
